spec/config_spec.rb

```diff
@@ -59,7 +61,7 @@ def check_keys(config)
     end
 
     it 'can create a new config from an existing config' do
-      config = Config.new(env: 'postlive')
+      config = Config.new(env: 'postlive', version: '11.1')
```

Comment:
Can we set this 12.1?

Reply:
Nvm it's for a spec. Disregard

Reply:
The reason I added this is because the config defaults to 12.1 now, so this test was failing because it expected it to be 11.1 at that point. So basically if we always want it to be 12.1, the test needs to be rewritten.